java/connector-node/risingwave-sink-jdbc/src/main/java/com/risingwave/connector/JDBCSink.java

```diff
@@ -70,7 +71,7 @@ public JDBCSink(JDBCSinkConfig config, TableSchema tableSchema) {
         try {
             var upsertSql =
                     jdbcDialect.getUpsertStatement(
-                            config.getTableName(),
+                            config.getNormalizedTableName(),
```

Comment:
I think the schema name would not be quoted by itself only, it should be quoted as `schema_name.table_name` for postgres-like databases.
FYI: https://nightlies.apache.org/flink/flink-docs-master/docs/connectors/table/jdbc/#jdbc-catalog

Reply:
For example, in the default implementation of `getInsertIntoStatement`, it will call `quoteIdentifier` on the table name, which may turn the `table_name` into a `"table_name"`. In this PR, the table name passed in is `schema_name.table_name`, and after quoted it becomes `"schema_name.table_name"`, while the expected one should be `"schema_name"."table_name"`. Therefore, we may have to pass the schema name in a separate parameter and call quoteIdentifier for `schema_name` and `table_name` separatedly.

Reply:
I totally understand your point. But my point is the expected one should be `"schema_name.table_name"` not the `"schema_name"."table_name"` after investigated some dialects. So it is unnecessary to quoted schema and table respectively.

Reply:
I think how to concatenate the schema name and table name is a database-specific behavior, and simply concatenating them with `.` is not the standard way to do it (at least in spark sql and presto it is `catalog`.`schema`.`table`). For better compatibility we'd better let the dialect to decide how to handle it and we should pass the original schema name and table name to the sql dialect. 

Maybe we can follow presto to introduce a class to include all table name related information and pass to the dialect instead of a simple String. https://github.com/prestodb/presto/blob/master/presto-spi/src/main/java/com/facebook/presto/spi/SchemaTableName.java

Reply:
> Maybe we can follow presto to introduce a class to include all table name related information and pass to the dialect

Sounds good to me.